### ISSUE TITLE
"DisabledUpload" bug

### ISSUE THREAD

**Issue by Speiger** [ShareX/ShareX#4681]
o/

You said its a duplicate to #4652
No its not. The setting you say was disabled by default and redisabling it won't fix the issue.

Any other way to fix it?

**Comment by Jaex** [comment]
Duplicate of #4679

**Comment by Speiger** [comment]
@Jaex could you at least give me a hint whats wrong because i want to continue using the software.

**Comment by Jaex** [comment]
Even you disabled upload application wide still you have usual upload options are enabled therefore ShareX is warning you.

**Comment by Speiger** [comment]
@Speiger no i had all the ones you showed in the "question" issue disabled.
I Fixed it though for myself now. Very simply: I installed a outdated version and will now no longer update for a long while until I think the issue is fixed from you.

So this is a new "Error Message" that was added. If this error appears then provide a "description" of how to disable this error or what the cause of this error is. You are only saying "Feature ???? is not working because upload disabled".

I would suggest you to fix the "error" reporting at least
